omod/src/main/java/org/openmrs/module/owa/web/controller/OwaManageController.java

```diff
@@ -51,12 +51,17 @@
 	}
 	
 	@RequestMapping(value = "/deleteApp", method = RequestMethod.GET)
-	public String deleteApp(@RequestParam("appName") String appName, ModelMap model) {
+	public String deleteApp(@RequestParam("appName") String appName, String owaName, ModelMap model) {
 		if (appName != null && Context.hasPrivilege("Manage OWA")) {
 			appManager.deleteApp(appName);
 			model.clear();
 		}
-		return "redirect:manage.form";
+
+		if (owaName == "restAPI") {
```

Comment:
What does "owaName" mean?

Reply:
@Charpell , how about using a more meaningful variable name for the optional param, like `requester` then check if(requester == "addonmanager")?

Reply:
@dkayiwa I have updated PR, "owaName" is a variable name for the optional param and i have used a more descriptive variable name.